extensions/cms/src/cmsUtils.ts

```diff
@@ -143,8 +143,6 @@ export class CmsUtils {
 		let toSaveCmsServers: ICmsResourceNodeInfo[] = this._registeredCmsServers.map(server => Object.assign({}, server));
 		toSaveCmsServers.forEach(server => {
 			server.ownerUri = undefined;
-			// don't save password in config
```

Comment:
if you remove this, the password will be clear text in the configuration file.

Reply:
Yeah, we can't do this. You'll need to persist the passwords in secret storage separately like I do in the BDC extension and then load it when you try to connect.

Saving : https://github.com/microsoft/azuredatastudio/blob/main/extensions/big-data-cluster/src/bigDataCluster/tree/controllerTreeDataProvider.ts#L154

Loading : https://github.com/microsoft/azuredatastudio/blob/main/extensions/big-data-cluster/src/bigDataCluster/tree/controllerTreeDataProvider.ts#L117